templates/default/systemd.service.erb

```diff
@@ -1,7 +1,7 @@
 <% prefix = @authbind ? '/usr/bin/authbind --deep ' : '' %>
 [Unit]
 Description = GlassFish Server for <%= @new_resource.domain_name %>
-After = syslog.target network.target remote-fs.target mysqld.service
+After = syslog.target network.target remote-fs.target mysqld.service <%= node['install']['systemd']['after'] %>
```

Comment:
Are you sure this works? chef-glassfish does not depends on conda-chef - I think this is giong to fail if the attribute is not set as you'll try to access some null values.

Reply:
That sounds correct. It might have passed testing due to some stale attributes in nodes.json file. I will test again with cleaning everything.